__tests__/index.js

```diff
@@ -1,39 +1,96 @@
 import config from "../"
 import stylelint from "stylelint"
-import test from "tape"
+import test from "ava"
 
-test("basic properties of config", t => {
-  t.ok(isObject(config.rules), "rules is object")
-  t.end()
-})
+const validCss = (
+`#selector-1,
+#selector-2,
+#selector-3 {
+	background: #fff;
+	color: #000;
+}
+
+#comment-form {
+
+	margin: 1em 0;
+}
+
+input[type="text"] {
+	line-height: 1.1;
+}
+
+.sample-output {
+	box-shadow: inset 0 0 1px 1px #eee;
+}
+
+.class { /* Correct usage of quotes */
+	background-image: url(images/bg.png);
+	font-family: "Helvetica Neue", sans-serif;
+}
+
+.class { /* Correct usage of zero values */
+	font-family: Georgia, serif;
+	text-shadow:
+		0 -1px 0 rgba(0, 0, 0, 0.5),
+		0 1px 0 #fff;
+}
+
+@media all and (max-width: 699px) and (min-width: 520px) {
 
-function isObject(obj) {
-  return typeof obj === "object" && obj !== null
+	/* Your selectors */
 }
 
-const css = (
+/**
+* #.# Section title
+*
+* Description of section, whether or not it has media queries, etc.
+*/
+
+.selector {
+	float: left;
+}
+
+a {
+	top: 0.2em;
+}
+`)
+
+const invalidCss = (
 `a {
-\ttop: .2em;
+	top: .2em;
 }
 
 `)
 
-stylelint.lint({
-  code: css,
-  config: config,
-})
-.then(checkResult)
-.catch(function (err) {
-  console.error(err.stack)
+test("no warnings with valid css", t => {
+  stylelint.lint({
+    code: validCss,
+    config: config,
+  })
+  .then(data => {
+    const { errored, results } = data
+    const { warnings } = results[0]
+    t.notOk(errored, "no errored")
+    t.is(warnings.length, 0, "flags no warnings")
+  })
+  .catch(function (err) {
+    console.error(err.stack)
+  })
```

Comment:
@TrySound Thanks! :)

@ntwb Are you able to update the tests in this PR if I go ahead and update `standard` and `suitcss`?